po/fr.po

```diff
@@ -0,0 +1,244 @@
+
+#: src/UI/MainWindow.vala:453
+msgid "Create a new empty file"
+msgstr "Créer un nouveau fichier vierge"
+
+#: src/UI/MainWindow.vala:469
+msgid "Save this file with a different name"
+msgstr "Enregistrer ce fichier avec un nom différent"
+
+#: src/UI/MainWindow.vala:477
+msgid "Redo"
+msgstr "Rétablir"
+
+#: src/UI/MainWindow.vala:485
+msgid "Undo"
+msgstr "Annuler"
+
+#: src/Widgets/StyleModal.vala:12
+msgid "Fonts"
+msgstr "Police"
+
+#: src/Widgets/StyleModal.vala:13
+msgid "Cells"
+msgstr "Cellule"
+
+#: src/Widgets/StyleModal.vala:30
+msgid "Color"
+msgstr "Couleur"
+
+#: src/Widgets/StyleModal.vala:34
+msgid "Set font color of a selected cell"
+msgstr "Définit la couleur de police de la cellule sélectionnée"
+
+#: src/Widgets/StyleModal.vala:38
+msgid "Reset font color of a selected cell to black"
+msgstr "Réinitialise la couleur de police de la cellule sélectionnée en noir"
+
+#: src/Widgets/StyleModal.vala:66
+msgid "Fill"
+msgstr "Remplissage"
+
+#: src/Widgets/StyleModal.vala:70
+msgid "Set fill color of a selected cell"
+msgstr "Définir la couleur de remplissage de la cellule sélectionnée"
+
+#: src/Widgets/StyleModal.vala:74
+msgid "Remove fill color of a selected cell"
+msgstr "Supprime la couleur de remplissage de la cellule sélectionnée"
+
+#: src/Widgets/StyleModal.vala:76
+msgid "Stroke"
+msgstr "Bordure"
+
+#: src/Widgets/StyleModal.vala:80
+msgid "Set stroke color of a selected cell"
+msgstr "Définit la couleur de bordure de la cellule sélectionnée"
+
+#: src/Widgets/StyleModal.vala:84
+msgid "Set the border width of a selected cell"
+msgstr "Définir la largeur de bordure de la cellule sélectionnée"
+
+#: src/Widgets/StyleModal.vala:88
+msgid "Remove stroke color of a selected cell"
+msgstr "Supprimer la couleur de border de la cellule sélectionnée"
+
+#: src/App.vala:23
+msgid "Add numbers"
+msgstr "Ajoute des nombres"
+
+#: src/App.vala:24
+msgid "Multiply numbers"
+msgstr "Multiplie des nombres"
+
+#: src/App.vala:25
+msgid "Divide numbers"
+msgstr "Divise des nombres"
+
+#: src/App.vala:26
+msgid "Substract numbers"
+msgstr "Soustrait des nombres"
+
+#: src/App.vala:27
+msgid "Gives the modulo of numbers"
+msgstr "Donne le modulo de deux nombres"
+
+#: src/App.vala:29
+msgid "Elevate a number to the power of a second one"
+msgstr "Met un nombre à la puissance d'un autre"
+
+#: src/App.vala:30
+msgid "The square root of a number"
+msgstr "La racine carrée d'un nombre"
+
+#: src/App.vala:31
+msgid "Rounds a number to the nearest integer"
+msgstr "Arrondi un nombre à l'entier le plus proche"
+
+#: src/App.vala:32
+msgid "Removes the decimal part of a number"
+msgstr "Retire la partie décimale d'un nombre"
+
+#: src/App.vala:33
+msgid "Return the smallest value"
+msgstr "Renvoie la plus petite valeur"
+
+#: src/App.vala:34
+msgid "Return the biggest value"
+msgstr "Renvoie la plus grande valeur"
+
+#: src/App.vala:35
+msgid "Gives the mean of a list of numbers"
+msgstr "Donne le sens d'une liste de nombres"
```

Comment:
In this context, "the mean" means "la moyenne" I think.

Reply:
Hmm you're right. I was not sure.

Reply:
Fixed in c643485